manifests/init.pp

```diff
@@ -167,14 +167,35 @@
     group   => $nagios_user,
   }
 
+  file { 'opsview-nagios-dir':
+    ensure  => directory,
+    path    => '/usr/local/nagios',
+    recurse => true,
+    mode    => '0550',
+    owner   => $nagios_user,
+    group   => $nagios_user,
+  }
+  file { 'opsview-libexec-dir':
+    ensure  => directory,
+    path    => '/usr/local/nagios/libexec',
+    recurse => true,
+    mode    => '0550',
+    owner   => $nagios_user,
+    group   => $nagios_user,
+    require => File['opsview-nagios-dir'],
+  }
+
+
   file { 'nrpe-scripts':
+    ensure  => directory,
```

Comment:
This seems wrong to me.

Reply:
Yeah, it's sucky. The "recurse" thingy does not work in the nrpe-scripts resource. It does not create /usr/local/nagios/libexec/nrpe_local's parents if they don't exist.